crates/torii/core/src/sql.rs

```diff
@@ -169,8 +169,8 @@ impl Sql {
         let keys_str = felts_sql_string(&keys);
         let insert_entities = "INSERT INTO entities (id, keys, event_id, executed_at) VALUES (?, \
                                ?, ?, ?) ON CONFLICT(id) DO UPDATE SET \
-                               executed_at=EXCLUDED.executed_at, event_id=EXCLUDED.event_id \
-                               RETURNING *";
+                               updated_at=CURRENT_TIMESTAMP, executed_at=EXCLUDED.executed_at, \
```

Comment:
Is it the only location where we should do this fix? Don't we also update them on change?

Reply:
We update them on change there too. We do an upsert which is what the ON CONFLICT is for